pkg/common-env.go

```diff
@@ -88,6 +88,8 @@ func PopulateInitParams(ctx goctx.Context) (*InitParams, error) {
 	workDir := asdbv1.GetWorkDirectory(rack.AerospikeConfig)
 	volume := rack.Storage.GetVolumeForAerospikePath(workDir)
 
+	tlsName, _ := asdbv1.GetServiceTLSNameAndPort(aeroCluster.Spec.AerospikeConfig)
```

Comment:
Now we are relying on `GetServiceTLSNameAndPort` to check tls or non-tls type, so should we remove `MY_POD_TLS_ENABLED` env from STS in operator?